concatenateHDF5.m

```diff
@@ -0,0 +1,50 @@
+function [hdf5Name] = concatenateHDF5(tifDir)
+% Concatenates all tif files in the specified directory into an
+% hdf5 file. A file structure similar to the one below should be used. The
+% stored files are all space binned by the factor set in 'downsmpFactor'.
+%
+% Example file structure: "G:\cohort9-herrin224\mouse5\day10_allo-south"
+% Example use: "concatenateHDF5(tifDir)" where 'tifDir' points to the
+%   folder containing the tif files
+%
+% 2015-01-06 Jessica Maxey
+
+cd(tifDir);
```

Comment:
Will this actually change the active directory in Matlab? If so, why not use `dir(fullfile(tifDir,'*.tif'))` which would list the tif files of the specified directory without changing the path?